README.md

```diff
@@ -85,13 +89,15 @@ To install cuDF from source, ensure the dependencies are met and follow the step
 ```bash
 CUDF_HOME=$(pwd)/cudf
 git clone https://github.com/rapidsai/cudf.git $CUDF_HOME
-cd CUDF_HOME
+cd $CUDF_HOME
 git submodule update --init --remote --recursive
 ```
 - Create the conda development environment `cudf_dev`
 ```bash
 # create the conda environment (assuming in base `cudf` directory)
-conda env create --name cudf_dev --file conda/environments/cudf_dev.yml
+conda env create --name cudf_dev --file conda/environments/cudf_dev.yml # for CUDA 9.2
+# or
+conda env create --name cudf_dev --file conda/environments/cudf_dev_cuda10.yml # for CUDA 10.0/1
```

Comment:
We don't support CUDA 10.1 yet so we don't want to advise users of that. Advise renaming to `cudf_dev_cuda10.0.yml`